server/auth/auth.py

```diff
@@ -7,11 +7,6 @@ class AuthTypeBase(ABC):
     def __init__(self):
         super().__init__()
 
-    @abstractmethod
-    def set_params(self, params):
-        """Set the parameters from app config.  raise ConfigurationError if any params are invalid"""
-        pass
-
     @abstractmethod
     def is_valid(self):
```

Comment:
Why not make this more explicitly named like `can_return_user_info` or something like that? `is_valid` has so many connotations in terms of auth in general so I had to search around to understand exactly what `is_valid` meant in the implementations.

Reply:
I don't quite like "can_return_user_info", since that might imply if can return other things.  I changed it to "is_valid_authentication_type".   Is that better?

Reply:
I also changed the names of a few other functions to make it more clear.
The ones with "user" in the name refer to the current user (or lack of current user): 
  is_user_authenticated, get_user_name, get_user_id, get_user_email.